FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/FhirToDataLakeOrchestratorJob.cs

```diff
@@ -150,40 +157,46 @@ await foreach (var input in inputs.WithCancellation(cancellationToken))
             catch (TaskCanceledException taskCanceledEx)
```

Comment:
Maybe OperationCanceledException is enough here? https://devblogs.microsoft.com/premier-developer/recommended-patterns-for-cancellationtoken/#handling-cancellation-exceptions

Reply:
Removed this exception.